releasenotes/notes/blacklist-payload-by-kind-65831a4a56776b32.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Add options to avoid some kind of payload from being sent to Datadog
+    For some user, basic information gathered by a check can be too sensitive
+    to be sent to Datadog (ex: IP addresses in events or service_check). We
+    now have options to blacklist any kind of payload from being sent to the
+    backend.
```

Comment:
Add options to exclude specific payloads from being sent to Datadog.  In some environments, some of the gathered information is considered too sensitive to be sent to Datadog (i.e. IP addresses in events or service checks).  This feature adds to option to exclude specific payload types from being sent to the backend.